frontera/contrib/backends/remote/codecs/json.py

```diff
@@ -101,52 +135,51 @@ def __init__(self, request_model, response_model, *a, **kw):
         super(Decoder, self).__init__(*a, **kw)
 
     def _response_from_object(self, obj):
-        url = to_native_str(obj[b'url'])
+        url = obj['url']
         request = self._request_model(url=url,
```

Comment:
> If url had type 'bytes' then after decoding it will have type bytes

You are right about the fact that this PR will save the type of the object it was originally given to the codec and on on decoding it will return  the original type of value.

> and in Frontera URLs have to be native strings

If this is the case than such type checking and conversion should be done somewhere else, Not In a codec. 

>  so maybe we still need a preventive conversion to native strings

A codec should never change the type of the object it was originally inputed with. Forcing such a mechanism in codec is a hacky way of getting around this problem.